production-v2.cfg

```diff
@@ -13,12 +13,19 @@
 # It is not a replacement for the production.cfg but an extension.
 
 [buildout]
-parts += setpermissions
+parts +=
+    fix-solr-permissions
```

Comment:
Yesterday we have agreed to put that in [`early-parts`](https://github.com/4teamwork/ftw-buildouts/blob/master/production.cfg#L3) so that it is executed _before_ the solr parts.

Why did you change that? Did you gain more insight?

Reply:
Yes it's the correct file.
But you registered the part with `parts +=` and not `early-parts +=`

Reply:
Ah, ok. I indeed did not get that part. I changed it now but like to test it before we merge.